stories/text/text.stories.js

```diff
@@ -0,0 +1,172 @@
+export default {
+    title: 'Components/Text',
+    parameters: {
+        description: `The text control is used to display text. It generally contains the text
+         that developers want apps to display (property: text).`,
+        tags: ['f3', 'a11y', 'theme'],
+        components: ['text']
+    }
+};
+
+export const defaultExample = () => `
```

Comment:
Remove newline at the end of this file to fix [empty jest snapshot](https://github.com/SAP/fundamental-styles/pull/1804/files#diff-4f34f45b3d2cc609fb28216e10a082a914ee15c29538f2d39a630b8b54af9994R3-R6).